tests/FunctionalTestCase.php

```diff
@@ -252,33 +260,62 @@ public static function getModuleInfo(string $row): ?string
     }
 
     /**
-     * Creates the test collection with the specified options.
+     * Creates the test collection with the specified options and ensures it is
+     * dropped again during tearDown(). If the collection already exists, it
+     * is dropped and recreated.
      *
-     * If the "writeConcern" option is not specified but is supported by the
-     * server, a majority write concern will be used. This is helpful for tests
-     * using transactions or secondary reads.
+     * A majority write concern is applied by default to ensure that the
+     * transaction can acquire the required locks.
+     * See: https://www.mongodb.com/docs/manual/core/transactions/#transactions-and-operations
+     *
+     * @param array $options CreateCollection options
      */
-    protected function createCollection(array $options = []): void
+    protected function createCollection(string $databaseName, string $collectionName, array $options = []): Collection
     {
-        $options += ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)];
+        // @see https://jira.mongodb.org/browse/PHPLIB-1145
```

Comment:
I asked this in another PR earlier today, but should this be `See: ` since we're not in a doc block? Fine either way as it's a matter of personal preference.

I suppose this makes it easier to grep for `@see` in the project.

Reply:
I notices the `See: ` in the phpdoc of this method. I think it's better to stick on 1 way to write comments.